src/script/conversation/ConversationRepository.ts

```diff
@@ -378,15 +376,14 @@ export class ConversationRepository {
     accessState?: string,
     options: Partial<NewConversation> = {},
   ): Promise<Conversation | undefined> {
-    const isFederated = Config.getConfig().FEATURE.ENABLE_FEDERATION;
-    const userIds = isFederated ? userEntities.map(user => user.qualifiedId) : userEntities.map(user => user.id);
+    const userIds = userEntities.map(user => user.qualifiedId);
 
     let payload: NewConversation & {conversation_role: string} = {
       conversation_role: DefaultRole.WIRE_MEMBER,
       name: groupName,
-      qualified_users: isFederated ? (userIds as QualifiedId[]) : undefined,
+      qualified_users: userIds,
       receipt_mode: null,
-      users: !isFederated ? (userIds as string[]) : [],
+      users: userIds.map(({id}) => id),
```

Comment:
cant you just map all of the userIds on line 379? 

`const userIds = userEntities.map(user => user.qualifiedId.id);`

Reply:
maybe im missing something

Reply:
I am using it as `QualifiedId[]` on line `384`. So I need both `QualifiedId[]` and `string[]` in the payload